src/ucp/rndv/proto_rndv.c

```diff
@@ -461,10 +461,11 @@ void ucp_proto_rndv_rts_query(const ucp_proto_query_params_t *params,
 
 void ucp_proto_rndv_rts_abort(ucp_request_t *req, ucs_status_t status)
 {
+    ucp_mem_h memh = req->send.state.dt_iter.type.contig.memh;
```

Comment:
are we sure the datatype is contig?

Reply:
Acquiring memh moved inside `memh_invalidate`.